Source/CombatExtended/CombatExtended/Comps/BipodComp.cs

```diff
@@ -133,6 +135,13 @@ public override void Notify_Unequipped(Pawn pawn)
 		{
 			IsSetUpRn = false;
 			ResetVerbProps(this.parent);
+			CanSetUp = false;
+		}
+
+
+		public override void Notify_Equipped(Pawn pawn)
+		{
+			CanSetUp = true;

```

Comment:
This is never cleared.  So when you start a game or load a save, it's false until the comp gets equipped, at which point it is true until the next game load.  

If you want to go this approach, CanSetUp needs to be set in notify_equipped, cleared in the uniquipped function, and stored/loaded on game save.  

A likely better approach is to make CanSetUp a getter, and have it check if the comp is held by a pawn dynamically.  It should be cheap enough to not need caching, but could be cached if it proves expensive.